build/tasks/prettier.js

```diff
@@ -0,0 +1,19 @@
+const { execSync } = require('child_process');
+
+module.exports = function (grunt) {
+  grunt.registerMultiTask(
+    'prettier',
+    'Task to run the prettier on generated files',
+    function () {
+      const files = grunt.task.current.data.files;
+
+      files.forEach(file => {
+        const src = Array.isArray(file.src) ? file.src : [file.src];
+
+        src.forEach(entry => {
+          execSync(`npx prettier --write ${entry}`, { stdio: 'inherit' });
+        });
```

Comment:
Why start a new process for each file?

```suggestion
        execSync(`npx prettier --write ${src.join(' ')`, { stdio: 'inherit' })
```

Or better yet, just use `require('prettier').format()` and keep everything in the same process.

Reply:
I tried, but It wouldn't run the correct parser on the files, it always tried to use `babel` and throw errors. I couldn't figure out how to load the proper config even with `.resolveConfig`